### PR TITLE
Add ping message

### PR DESCRIPTION
**Pull request by cjellick**
Add a periodic ping message to act as a keep alive between proxy and
host-api.

### CODE REVIEW

File: proxy/multiplexer.go [hunk 1]
```diff
@@ -83,6 +87,7 @@ func (m *multiplexer) routeMessages(ws *websocket.Conn) {
 
 	// Write messages to backend
 	go func(stop <-chan bool) {
+		c := time.Tick(time.Second * 5)
```
Comment (ibuildthecloud) [inline proxy/multiplexer.go]: will this leak goroutinues over time?

Comment (ibuildthecloud) [inline proxy/multiplexer.go]: Don't know the context of this code, if this run per connection or once

Comment (cjellick) [inline proxy/multiplexer.go]: @ibuildthecloud no, this wont leak goroutines. This goroutine is the the loop that send messages from the proxy to host api. So, there is one per backend connection and when the backed connection is closed, utlimately either the `messagesToBackend` channel is closed or a message comes in on the `stop` channel, the for loop exits, and the goroutine returns.

...but none of that is new logic. This adds a ping every 5 seconds to the select inside of the for loop. So, unless I'm missing the point of your question, I can't see how this change would introduce a potential goroutine leak.

Comment (ibuildthecloud) [inline proxy/multiplexer.go]: @cjellick I was actually thinking about the Tick.  That creates a goroutine, right?  That can never be closed??

Comment (ibuildthecloud) [inline proxy/multiplexer.go]: Oh wait, there's a c.Stop() method.  Should you call that?

Comment (cjellick) [inline proxy/multiplexer.go]: Yes.
How embarrassing.

**Pull request merged**